### DISCUSSION TITLE
How to get MVCObject from marker? I want to put InfoWindow on marker.

### DISCUSSION THREAD

**Discussion by spencerchang** [JustFly1984/react-google-maps-api#3206] [Q&A]
InfoWindow's props position is not good enough and i want to use props anchor but it accepts MVCObject type.
How do i get or create a MVCObject type object to  put InfoWindow on marker?

Thank you.

**Comment by crtl** [comment]
Use [`OverlayView`](https://react-google-maps-api-docs.netlify.app/#overlayview) to position overlay relative to marker.
